Penumbra/Configuration.cs

```diff
@@ -87,6 +87,8 @@ public class Configuration : IPluginConfiguration, ISavable
     public Dictionary<ColorId, uint> Colors { get; set; }
         = Enum.GetValues<ColorId>().ToDictionary(c => c, c => c.Data().DefaultColor);
 
+    public IReadOnlyList<string> SharedTags { get; set; }
```

Comment:
I think I want the tags in a separate file rather than the main config file. This is a bunch more work, of course, but I very much prefer multiple smaller config files. You'll need to implement ISavable, can probably look at ephemeral_config.json or something to start with. You can also tell me to do it, but for now I treat this as a review.
A file version should be included, even if it is unlikely that the format changes later on (or changes in a way that doesn't just add stuff).

Reply:
I'll take a shot at splitting this out into a separate config.